django/urls/resolvers.py

```diff
@@ -62,10 +62,14 @@ def __repr__(self):
         )
 
 
-@functools.lru_cache(maxsize=None)
 def get_resolver(urlconf=None):
```

Comment:
When this would get called with `None` (the default) in one place, and `settings.ROOT_URLCONF` in another, you'd end up with two separate `URLResolver` objects.